app/src/main/java/com/example/guineapics/dao/GuineaPigsDAO.java

```diff
@@ -6,6 +6,18 @@
 
 public class GuineaPigsDAO implements DataAccessable<GuineaPig> {
```

Comment:
what is DataAccessable? 🤨

Reply:
It's a DAO interface. Should I name it differently?